docker/environment.yml

```diff
@@ -0,0 +1,15 @@
+name: env
```

Comment:
This file is not used for now, either use it or remove it.
I'd advise using it, but this may take a bit more testing than just removing it, so you may just remove it.